index.js

```diff
@@ -322,9 +322,11 @@
 
     return {
       prefix: prefix,
-      value: power === undefined
+      value: (
+        power === undefined
         ? value / factor
         : Math.round(value * power / factor) / power
+      ).toFixed(decimals)
```

Comment:
Thanks for your contribution :tada: 

`value` here is number, please don't change this.

You may want to apply to the function `humanFormat` instead :slightly_smiling_face: 

Also, maybe we could keep the old behavior by having both a `decimals` (new) and a `maxDecimals` (old) options?